tests/interfaces.py

```diff
@@ -832,6 +884,72 @@ def test_UpdateUsers_username_Forbidden_UpdateOthers(self):
         resp = utils.test_request(self, "GET", path, cookies=self.cookies, headers=self.json_headers)
         utils.check_response_basic_info(resp)
 
+    @runner.MAGPIE_TEST_USERS
+    def test_DeleteUser_DeleteSelf(self):
+        """
+        Logged user is allowed to delete his own account.
+        """
+        # login as test user and delete its own account
```

Comment:
add skip safeguard, 
other tests are ok because 403 remains 403.
but this one is a change of behavior

Reply:
Should I also add a skip safeguard to the corresponding test on the admin scope? Or was this an existing behavior for the admin level?

https://github.com/Ouranosinc/Magpie/blob/859340a9771b1995212b8243e12dc8444212a2f2/tests/interfaces.py#L3088-L3093

Reply:
The admins were allowed to self delete before as they can do anything (except remove special anonymous/admin).
So the result should be the same.

Reply:
Ok! I won't put a safeguard for the admin case then.